source/_integrations/doorbird.markdown

```diff
@@ -19,13 +19,12 @@ There is currently support for the following device types within Home Assistant:
 
 ## Setup
 
-It is recommended to set up a new account on your Doorbird for use with Home Assistant. This can be added via the Doorbird App by choosing Administration -> (User) Add. This user, needs specific permissions enabled, depending on what functionality you want:
+It is recommended to setup a new account on your Doorbird App for use with Home Assistant. This can be added via the Doorbird App by clicking settings (cog icon) -> Administration-> LOGIN (using your App Administration details). Under the "USER" section, choose "Add". This new user account requires specific permissions enabled (depending on what functionality you want). Permissions can be found under "Permissions". The following permissions are recommended:
```

Comment:
`setup` -> `set up`. That change is invalid. `setup` is a noun, which is arranged already. `set up` is a verb to tell the user to take action.

Please revert that.

```suggestion
It is recommended to set up a new account on your Doorbird App for use with Home Assistant. This can be added via the Doorbird App by clicking settings (cog icon) -> Administration-> LOGIN (using your App Administration details). Under the "USER" section, choose "Add". This new user account requires specific permissions enabled (depending on what functionality you want). Permissions can be found under "Permissions". The following permissions are recommended:
```

Reply:
Thanks @frenck ... too many years of clicking setup.exe!